ex_2.js

```diff
@@ -0,0 +1,38 @@
+// Exercitiu 2: Scrieți o funcție care returnează un obiect nou fără proprietățile furnizate
+
+//Funcția primește 2 parametri ca date de intrare. Unul dintre parametri este obiectul, iar celălalt este proprietatea care nu se dorește să persiste în rezultatul returnat.
+
+//Exemplu 1:
+
+//Input: obiect = { a: 1, b: 2 }, proprietate = “b”
+
+//Output: { a: 1 }
+//
+
+
+function removeProperties(car, model) {
```

Comment:
```suggestion
function removeProperties(object, properties) {
```

Functia e facuta sa mearga global - nu doar pentru obiectul "car". Iar al doilea parametru se asteapta o lista de parametri (asa cum ai facut tu), nu doar o proprietate "model". Duce in confuzie